bindings/nodejs/types/secretManager.ts

```diff
@@ -1,3 +1,7 @@
+export interface LedgerNanoSecretManager{}
+
+export interface LedgerNanoSimulatorLedgerNanoSecretManager{}
```

Comment:
Doesn't really make sense to do this, since both are the same interface (empty object). Furthermore empty interfaces are not good design practice, since they work differently than in Rust. Maybe an option would be to provide a wrapper interface that contains a boolean for `isSimulator` property?

Reply:
As an alternative, you could define a `Ledger` enum with two possible values (`Simulator/Hardware`).

Reply:
Opened a PR in iota.rs to remove the simulator variant and add the `isSimulator` bool